enterprise/pkg/a8n/store.go

```
@@ -794,6 +795,8 @@ func scanChangeset(t *a8n.Changeset, s scanner) error {
 	switch t.ExternalServiceType {
 	case github.ServiceType:
 		t.Metadata = new(github.PullRequest)
+	case bitbucketserver.ServiceType:
+		t.Metadata = new(bitbucketserver.PullRequest)
 	default:
 		return nil
```

Should we return an error here?

---

Done!